src/cloudlabs/templates/includes/_dashboard.html

```diff
@@ -0,0 +1,50 @@
+<!-- Listing of current user's machines -->
+
+<p>
+  This is your dashboard where you can see the list of machines you have created and their status,
+  as well as stop/start them, make alterations to their configuration, download local copies, and create new machines.
+</p>
+
+<table class="hosts">
+  <tr>
+      <th>Name</th>
+      <th>Address</th>
+      <th>Status</th>
+      <th>Actions</th>
+  </tr>
+  {# 
+    TODO: If we're calling a REST service to get this, this will become JS code,
+    in which case we'll need to figure out how to do url_for in JS...
+  #}
+  {% for host in user.hosts %}
+    <tr>
+      <td>{{ host.label }}</td>
+      <td><a href="{{ host.link }}">{{ host.basic_url }}</a></td>
+      <td>{{ host.status }}</td>
+      <td>
+        {% if host.status in ['Running', 'Restarting'] %}
+          <a href="{{ url_for('host.control', id=host.id, action='stop') }}"><img src="{{ url_for('static', filename='img/stop.svg') }}" alt="Stop virtual machine" class="action" /></a>
+        {% endif %}
+        {% if host.status == 'Running' %}
+          <a href="{{ url_for('host.control', id=host.id, action='restart') }}"><img src="{{ url_for('static', filename='img/restart.svg') }}" alt="Restart virtual machine" class="action" /></a>
+        {% endif %}
+        {% if host.status == 'Stopped' %}
+          <a href="{{ url_for('host.control', id=host.id, action='start') }}"><img src="{{ url_for('static', filename='img/start.svg') }}" alt="Start virtual machine" class="action" /></a>
+        {% endif %}
+        {# TODO: Add a 'Deploying' status with action to view log #}
+        <a href="{{ url_for('host.edit', id=host.id) }}"><img src="{{ url_for('static', filename='img/configure.svg') }}" alt="Configure virtual machine') }}" class="action" /></a>
+        <a href="{{ url_for('host.download', id=host.id) }}"><img src="{{ url_for('static', filename='img/download.svg') }}" alt="Download virtual machine image" class="action" /></a>
+        <img src="{{ url_for('static', filename='img/delete.svg') }}" alt="Delete virtual machine" class="action" onclick="confirm_delete('{{ url_for('host.delete', id=host.id) }}')" />
+        <a href="{{ url_for('host.info', id=host.id) }}"><img src="{{ url_for('static', filename='img/info.svg') }}" alt="Show virtual machine info" class="action" /></a>
```

Comment:
Is the info button supposed to show the description of the item? Or is that at a later stage?